src/Symfony/Component/Config/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+4.4.0
+-----
+
+ * added a way to exclude patterns of resources from being loaded
```

Comment:
This sentence is too generic. You should mention the method and the new argument.

Reply:
I improved it. I'll mark this remark as solved. Don't hesitate to re-open it if it must be improved again. Thank you.